app/controllers/shops_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ShopsController < ApplicationController
+    def index
+        @shops = Shop.all
```

Comment:
Nice! at some point you may decide to bring this over to a React view instead of ERB